packages/runtime-dom/src/helpers/useCssVars.ts

```diff
@@ -57,7 +57,15 @@ function setVarsOnVNode(
   if (vnode.shapeFlag & ShapeFlags.ELEMENT && vnode.el) {
     const style = vnode.el.style
     for (const key in vars) {
-      style.setProperty(`--${prefix}${key}`, unref(vars[key]))
+      if (typeof vars[key] === 'string') {
+        style.setProperty(`--${prefix}${key}`, unref(vars[key]))
+      } else {
+        console.warn(
```

Comment:
This should be a dev only warn using the existing warn function

Reply:
You are quite correct, I should have realized that since warn() is already in use in the very same file. I'll change that.